gk/bpf.c

```diff
@@ -511,5 +517,14 @@ gk_bpf_decide_pkt(struct gk_config *gk_conf, uint8_t program_index,
 	*p_bpf_ret = likely(handler->f_pkt_jit != NULL)
 		? handler->f_pkt_jit(&frame.ctx)
 		: rte_bpf_exec(handler->f_pkt, &frame.ctx);
+
+	if (unlikely(*p_bpf_ret == GK_BPF_PKT_RET_FORWARD &&
+			frame.called_encapsulate == false)) {
```

Comment:
Same thing here with explicitly comparing against a boolean literal -- looks a little weird.

Reply:
I'll drop it. It's just that DPDK's code has a preference to compare the actual values instead of using implicit tests. But reviewing [1.8.1. NULL Pointers](https://doc.dpdk.org/guides/contributing/coding_style.html#c-statement-style-and-conventions) of the coding style, I see it's okay with `bool` values.